xrpl/core/binarycodec/definitions/definitions.json

```diff
@@ -57,7 +57,8 @@
     "Child": -2,
     "Nickname": 110,
     "Contract": 99,
-    "GeneratorMap": 103
+    "GeneratorMap": 103,
+    "Credential": 129
```

Comment:
nit: should go after `Oracle`

Reply:
ok, I've updated in f476a51e
It doesn't look like there is any rationale for the order of LedgerEntryTypes -- alphabetical or numeric.